markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+      \begin{verbatim}
+        Christoph Lakner World Bank ORCID 0000, clakner@worldbank.org
+      \end{verbatim}
+    \item Contributors: like RAs (not conference participants). Also coeditors if specifically mentioned. FORMAT
+    \begin{lstlisting}
+      Coeditor
+    \end{lstlisting}
+    \item Sponsors: All financial contributions and individual/general grants
+    \item Acknowledgements: Usually empty; Example: Corporate Taxation: special mentioning ministry employees for data assistance (could go to acknowledgment statement)
+    \item Acknowledgement Statement:
+      \begin{verbatim}
+        We are grateful to Benu Bidani, Francisco Ferreira, Luis-Felipe..
+      \end{verbatim}
+      \begin{verbatim}
+        .. and seminar participants at the World Bank’s Poverty ...
+      \end{verbatim}
+  \end{enumerate}
+
+## Version statement
+Sufficient are
+\begin{enumerate}
+    \item Project completion date: FORMAT: YEAR-MONTH
+      \begin{itemize}
+        \item PP: Date at which the article was published: Year-Month
+        \item WP: \textcolor{red}{Must be supplied, bc it is needed for the front page
+        of cataloge}
+      \end{itemize}
+    \item Version date: FORMAT: YEAR-MONTH-DAY
+      \begin{itemize}
+        \item PP: empty
+        \item WP: data of publication as research paper in WB Year-Month-Day
+      \end{itemize}
+\end{enumerate}
+
+## Scope and Coverage
+Sufficient are
+\begin{enumerate}
+    \item Abstract: Abstract of the article; \textcolor{red}{Hyphens are a problem. Check all abstracts. Also carriage returns are a problem, direct copy pasting leads to bad results}
+    \item Geographic areas: Format: Name, Code, Type
+    \begin{lstlisting}
+      Lebanon, LBN, ISO-3166 Alpha-3
+    \end{lstlisting}
+    \item Keywords: No keywords if none are given; Yes if given.
+    \item JEL codes:
+      \begin{verbatim}
+        ID D63
+        Name Equity, Justice, Inequality
+        Parent ID (empty)
+        Vocabulary JEL Classifications
+        URL https://www.aeaweb.org/econlit/jelCodes.php?view=jel
+      \end{verbatim}
+\end{enumerate}
+
+## Processes and Output
+The following fields exist
+\begin{enumerate}
+    \item Process: empty for PP, tbd for WP
+    \item Institutional review board: empty
+    \item Approval process: empty
+    \item Reviews and comments:
+      \begin{itemize}
+        \item WP: If available (tbd, 2023-06-22)
+        \item PP: empty
+      \end{itemize}
+    \item Output: 
+        \begin{itemize}
+          \item Type    
+            \begin{itemize}
+              \item WP: Working Paper
+              \item PP: Published Paper (not paper, not article, not Article)
+            \end{itemize}
+          \item Title: Title of the paper
+          \item Authors: Authors: all comma separated; first name, last name. Patty Smith, Patty Smith, Petty Smith
```

Comment:
isn't this repeating the information from authoring entity? why do we need it in both places?

Reply:
We did it like this when Olivier showed me the editor. I do not mind it: True we replicate the information, but it is very nice to have all information on the output drop down